.github/workflows/formal-verification.yml

```
@@ -66,3 +67,20 @@ jobs:
           node certora/run.js ${{ steps.arguments.outputs.result }} >> "$GITHUB_STEP_SUMMARY"
         env:
           CERTORAKEY: ${{ secrets.CERTORAKEY }}
+
+  halmos:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up environment
+        uses: ./.github/actions/setup
+      - name: Install python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PIP_VERSION }}
+          cache: 'pip'
+          cache-dependency-path: 'fv-requirements.txt'
+      - name: Install python packages
+        run: pip install -r fv-requirements.txt
+      - name: Run Halmos
+        run: halmos --match-test '^check_|^testSymbolic' -vv
```

The `check_` pattern requires adding `solhint-disable func-name-mixedcase` in many test files. Why not use a different prefix. I'm proposing:
```suggestion
        run: halmos --match-test '^checkSymbolic|^testSymbolic' -vv
```

---

Another, potentially better, option would be to have:
- `testXxxxxx` for foundry fuzzing test
- `symbolicXxxxxx` for halmos

So that there is no overlap, and everything is clear.